src/promptflow/promptflow/core/_serving/flow_invoker.py

```diff
@@ -75,14 +76,21 @@ def _init_connections(self, connection_provider):
             config = {"connection.provider": connection_provider} if connection_provider else None
             self.logger.info(f"Getting connections from pf client with provider from args: {connection_provider}...")
             connections_to_ignore = list(self.connections.keys())
+            self.logger.debug(f"Flow invoker connections: {self.connections.keys()}")
             connections_to_ignore.extend(self.connections_name_overrides.keys())
+            self.logger.debug(f"Flow invoker connections name overrides: {self.connections_name_overrides.keys()}")
+            self.logger.debug(f"Ignoring connections: {connections_to_ignore}")
```

Comment:
what's this scenario?

Reply:
Offline synced, used for connection object override in flow-as-a-func